js/src/OrdinalScaleModel.ts

```diff
@@ -27,7 +27,7 @@ export class OrdinalScaleModel extends ScaleModel {
   }
 
   typedRange(values) {
-    return new Int32Array(values.map(Number));
+    return new Int32Array(values.map((value) => this.domain.indexOf(value)));
```

Comment:
could it just be:
```ts
return new Int32Array(values.map(this.domain.indexOf));
```
this would avoid allocating a new function/scope each time... though maybe the `this` shenanigans come into play...

Reply:
I originally tried that but it was throwing errors... probably due to `this` shenanigans.

Reply:
k, how about:

```ts
return new Int32Array(values.map(this.domain.indexOf, this.domain));
```

Reply:
Yes that behaves and pushed the change.